packages/core/src/accessanalyzer/vue/iamPolicyChecks.ts

```diff
@@ -140,7 +423,72 @@ async function _getReferenceDocument(input: string): Promise<string> {
             const s3Client = new S3({ region })
             const resp = await s3Client.getObject({ Bucket: bucket, Key: key }).promise()
             return resp.Body!.toString()
-        } catch (e: any) {}
+        } catch (e: any) {
+            if (e.message.includes('Invalid S3 URI')) {
+                throw new PolicyChecksError('Invalid file path or S3 URI', PolicyChecksErrorCode.FileReadError)
+            } else {
+                throw new PolicyChecksError(e.message, PolicyChecksErrorCode.FileReadError)
+            }
+        }
     }
-    return ''
+}
+
+//Check if Cfn and Tf tools are installed
+function arePythonToolsInstalled(): boolean {
```

Comment:
Some logging about which tool isn't found may be helpful.

Reply:
Added logging for which tool isnt found.